### PR TITLE
Adding function with max value

### PR DESCRIPTION
**Pull request by Kasia9311**
Adding function with max vector value

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @Kasia9311

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Kasia9311 for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -4,5 +4,12 @@
 
 int maxOfVector(const std::vector<int>& vec) {
     // TODO: Implement me :)
-    return {};
+    int temp {std::numeric_limits<int> :: min()};
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:7]: operator `::` should not have spaces. Check `std::vector`, `std::cout` :)

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -4,5 +4,12 @@
...
+    int elem {};
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:8]: Elem is redundant here.

File: homework/max-of-vector/maxOfVector.hpp [hunk 3]
```diff
@@ -4,5 +4,12 @@
...
+
+    for (const auto& elem : vec) {
+        if (elem > temp) {
+            temp = elem;
+        }
+    } return temp;
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:14]: Please move return to new line.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**